Gemfile.lock

```diff
@@ -336,7 +336,7 @@ GEM
       msgpack
     debug_inspector (0.0.3)
     diffy (3.4.2)
-    dogstatsd-ruby (3.3.0)
+    dogstatsd-ruby (5.6.1)
```

Comment:
### [dogstatsd-ruby](https://github.com/DataDog/dogstatsd-ruby)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 3.3.0 → 5.6.1

[_[change-log](https://github.com/DataDog/dogstatsd-ruby/blob/v5.6.1/CHANGELOG.md), [source-code](https://github.com/DataDog/dogstatsd-ruby/tree/v5.6.1), [gem-diff](https://my.diffend.io/gems/dogstatsd-ruby/3.3.0/5.6.1)_]





<details>
<summary>Commits</summary>

A change of **401** commits. See the full changes on [the compare page](https://github.com/DataDog/dogstatsd-ruby/compare/v3.3.0...v5.6.1).

These are the first 10 commits:
- (15d07c3) [Restores kbogtob performances improvements (#151 and #146)](https://github.com/DataDog/dogstatsd-ruby/commit/15d07c341bfe6c0027c6edbea7c577926ec9cd0c)
- (79ce420) [Introduce truncate_if_too_long option to allow users to send…](https://github.com/DataDog/dogstatsd-ruby/commit/79ce4201743b34dd89abd5d9565e0964d8c5685c)
- (c7f1f79) [Comment the new event method option](https://github.com/DataDog/dogstatsd-ruby/commit/c7f1f79baca3921e2cbc4ccb41ea68ae6d784817)
- (958b5e0) [Merge pull request #161 from kazu9su/master](https://github.com/DataDog/dogstatsd-ruby/commit/958b5e0be075d2b132c45a70aac01e7ede8b2d30)
- (4065445) [Release 4.8.3](https://github.com/DataDog/dogstatsd-ruby/commit/406544560f42c0ca3adc1bdaa5cd72d7869d2c35)
- (94c62c6) [[Gemfile] Update rack dep (#152)](https://github.com/DataDog/dogstatsd-ruby/commit/94c62c661872c5e32fc2d0b902c4b55e9d3d514a)
- (eb5a9b3) [Merge pull request #165 from DataDog/remeh/4.8.3](https://github.com/DataDog/dogstatsd-ruby/commit/eb5a9b39b3cc8ff69499bc22cfabd6ce0e189976)
- (3ce9122) [Fix typo](https://github.com/DataDog/dogstatsd-ruby/commit/3ce912294c7b2a2d9cc3ace116f33307518ad8eb)
- (2e07024) [Merge pull request #166 from vecerek/patch-1](https://github.com/DataDog/dogstatsd-ruby/commit/2e07024486f9f5d7a13d92007f161732cf4e7d14)
- (5a7b40e) [Merge branch 'master' of github.com:DataDog/dogstatsd-ruby i…](https://github.com/DataDog/dogstatsd-ruby/commit/5a7b40e8825ac3afc31b23e69965af618cb73685)


</details>